lib/utils.pm

```diff
@@ -258,6 +259,23 @@ sub get_netboot_mirror {
     return get_var('MIRROR_' . uc($m_protocol));
 }
 
+sub zypper_call {
+    my ($command, $order) = @_;
+    $order = $order // 1;
+    script_run("zypper -n $command; echo worked-patch-\$?- > /dev/$serialdev", 0);
+    my $ret = wait_serial qr/worked-patch-\d+-/, 2000;
+
+    if ($ret =~ /worked-patch-(\d+)/) {
+        if ($order == 2) {
```

Comment:
is really needed zypper_calll function ...?


```
script_run("zypper -n patch --with-interactive -l; echo worked-patch-\$? > /dev/$serialdev", 0)
wait_serial( qr/worked-patch-(0|102|103)-/,700)
```

will be enough ? @coolo @aaannz